### PR TITLE
[virtual env] release the conda version constraints

### PR DESCRIPTION
**Pull request by scv119**
## Why are these changes needed?

conda has released several versions. see if the problem still exists in latest version.

previous issue: https://github.com/ray-project/ray/pull/34650/files

## Checks

- [ ] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
    - [ ] I've added any new APIs to the API Reference. For example, if I added a 
           method in Tune, I've added it in `doc/source/tune/api/` under the 
           corresponding `.rst` file.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Review by rkooo567 — APPROVED** [review]
didn't we have a issue related to it? Should we close it?

File: python/ray/_private/usage/usage_lib.py [hunk 1]
```diff
@@ -569,6 +569,40 @@ def get_extra_usage_tags_to_report(gcs_client) -> Dict[str, str]:
     return extra_usage_tags
 
 
+def _get_cluster_status_to_report_v2(gcs_client) -> ClusterStatusToReport:
```
Comment (jjyao) [inline python/ray/_private/usage/usage_lib.py:572]: ?

**Comment by anyscalesam** [comment]
LiveEO needs this and we've set that exepctation see here https://anyscaleteam.slack.com/archives/C04UR8J709K/p1701099640949819

**Review by richardliaw — APPROVED** [review]

**Review by aslonnie — COMMENTED** [review]
PR is merge-able after synced to master head.

the virtualenv on the CI might not be actually changed, as in most places they are pinned, and the pin is not changed.

so you would need to make a judgement yourself if that is the intended behavior / if it is properly tested. The pinned virtualenv version is the old version and still satisfies the new requirement.

**Pull request merged**